### PR TITLE
fix russian translate

### PR DESCRIPTION
**Pull request by almirus**

### CODE REVIEW

**Comment by ixuz** [comment]
Thank you @almirus I will review these changes shortly

File: SidebarDiagnostics/Properties/Resources.ru.resx [hunk 1]
```diff
@@ -558,7 +558,7 @@
     <comment>Settings background color tooltip</comment>
   </data>
   <data name="SettingsBackgroundOpacity" xml:space="preserve">
-    <value>Прорачность фона</value>
+    <value>Прозрачность фона</value>
```
Comment (ixuz) [inline SidebarDiagnostics/Properties/Resources.ru.resx:561]: The translation of this should probably be "непрозрачность фона".
* An opacity value of 0% is effectively fully transparent.
* An opacity value of 100% is effectively fully opaque.

Do you agree?

Comment (almirus) [reply SidebarDiagnostics/Properties/Resources.ru.resx:561]: yep! "Непрозрачность фона"

Comment (ixuz) [reply SidebarDiagnostics/Properties/Resources.ru.resx]: 👍

**Comment by ixuz** [comment]
Thank you @almirus for your contribution

**Comment by ArcadeRenegade** [comment]
Published in version 3.5.6

**Pull request merged**